docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -202,11 +194,6 @@ To install the newest version, you may need to unmask the ansible package prior
 
     $ echo 'app-admin/ansible' >> /etc/portage/package.accept_keywords
 
-.. note::
-
-   If you have Python 3 as a default Python slot on your Gentoo nodes (default setting), then you
-   must set ``ansible_python_interpreter = /usr/bin/python2`` in your group or inventory variables.
```

Comment:
Gentoo defaults to  3.4, so we might want to leave the note but change the content, but probably make it 'some distros' as it is not the only one.